src/Components/MasonryLayout/MasonryBox/MasonryBox.module.css

```
@@ -25,45 +28,75 @@
   justify-content: center;
   align-items: center;
   z-index: 999;
+  padding: 0 30px;
+}
+
+.enlargedPhotoContainer {
+  display: flex;
+  background: url(https://raw.github.com/mmoustafa/Chalkboard/master/img/bg.png);
+  width: 80%;
+  height: 70%;
+  border-radius: 10px;
 }
 
 .enlargedPhotoImage {
-  max-width: 50%;
-  max-height: 50%;
+width: 50%;
+object-fit: cover;
+border-radius: 10px 0px 0px 10px
+  /* max-height: 50%;  */
 }
 
 .enlargedPhotoText {
-  color: black;
+  overflow-y: scroll;
```

Setting the:
`overflow-y: auto;`
```suggestion
  overflow-y: auto;
```
will eliminate the side scroll bar that is currently popping up (see picture below and this sidebar currently when a contributor text is short, will be displayed weirdly. I believe it is best for now to set to `auto` for simplicity. Let me know your thoughts if you have another solution to prevent the scroll bar from moving inwards on the text box when a contributor's text is short.